app/components/articleSearch/components/searchItem/abstract.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+const styles = require("./abstract.scss");
+
+export interface IAbstractProps {
+  abstract: string;
+  isAbstractOpen: Boolean;
+  toggleAbstract: () => void;
+}
+
+const Abstract = (props: IAbstractProps) => {
+  if (props.abstract === null) return null;
+  // for removing first or last space
+  const trimmedAbstract = props.abstract.replace(/^ /gi, "");
+  const restParagraphStartIndex = trimmedAbstract.indexOf("\n");
+
+  // if (props.isAbstractOpen) {
```

Comment:
Remove unused lines

Reply:
I will use this line after connecting with Social Auth
So I remained this.